src/test/com/fulcrologic/rad/database_adapters/sql_test.clj

```diff
@@ -0,0 +1,70 @@
+(ns com.fulcrologic.rad.database-adapters.sql-test
+  (:require
+    [com.fulcrologic.rad.attributes            :as rad.attr :refer [defattr]]
+    [com.fulcrologic.rad.database-adapters.sql :as rad.sql]
+    [clojure.test :refer [deftest testing is]]))
+
+(defattr account-id :account/id :uuid
+  {::rad.attr/identity? true
+   ::rad.sql/schema :production
+   ::rad.sql/tables #{"accounts"}})
+
+(defattr account-name :account/name :string
+  {::rad.sql/schema :production
+   ::rad.sql/tables #{"accounts"}})
+
+;; Derived data
+(defattr account-locked? :account/locked? :boolean
+  {})
+
+(defattr user-id :user/id :uuid
+  {::rad.sql/schema :production
+   ::rad.sql/tables #{"users"}})
+
+(defattr user-name :user/name :string
+  {::rad.sql/schema :production
+   ::rad.sql/tables #{"users"}})
+
+(deftest delta->txs-test
+  (with-redefs [rad.attr/key->attribute {:account/id      account-id
```

Comment:
I am generally trying to get rid of the registry, FYI.

Reply:
yeah I'd like the attrs to be passed down and used without global registry. Would simplify any test, and makes more sense. I know what to do.